packages/cf-component-dropdown/src/Dropdown.js

```diff
@@ -81,4 +128,4 @@ Dropdown.childContextTypes = {
   dropdownRegistry: PropTypes.instanceOf(DropdownRegistry).isRequired
 };
 
-export default Dropdown;
+export default createComponent(styles, Dropdown, ['align']);
```

Comment:
Does it? I thought I always had to specify this 😱 or is that just when it's not part of the `propTypes`?